vipps-ecom-api-faq.md

```diff
@@ -602,6 +603,25 @@ See:
 
 ## Users and payments
 
+### When do users get a "soft decline" and need to complete a 3-D Secure verification?
+
+Vipps uses delegated SCA (Secure Customer Authentication) from the banks, and
+significantly simplifies the user experience, as there is usually no need for
+BankID verification. The biometric login in Vipps is enough.
+
+Vipps uses tokenized cards, which eliminates the need for "soft decline".
+As long as the token is valid, the user never has to verify the card again.
+
+Due to rules and regulations Vipps _is_ required to do a 3-D Secure
```

Comment:
this is a risk check that vipps has imposed on itself, we are not required, but we choose to

Reply:
Thanks!